firmware/application/usb_serial_shell.cpp

```diff
@@ -1053,6 +1053,84 @@ static void cmd_settingsreset(BaseSequentialStream* chp, int argc, char* argv[])
     chprintf(chp, "ok\r\n");
 }
 
+static void cmd_sendpocsag(BaseSequentialStream* chp, int argc, char* argv[]) {
+    const char* usage = "usage: sendpocsag <addr> <msglen> [baud] [type] [function] [phase] \r\n";
+    (void)argv;
+    if (argc < 2) {
+        chprintf(chp, usage);
+        return;
+    }
+    uint64_t addr = atol(argv[0]);
+    uint8_t msglen = atoi(argv[1]);  // without minimum limit, since addr only don't send anything
```

Comment:
what if I enter 123456789 as a msglen ? Isn't it going to gravely overflow ?

Reply:
changed to int, and checking for <0 too